darts/timeseries.py

```diff
@@ -140,156 +582,251 @@ def pd_dataframe(self, copy=True) -> pd.DataFrame:
         Returns
         -------
         pandas.DataFrame
-            The Pandas Dataframe underlying this time series
+            The Pandas DataFrame representation of this time series
         """
+        self._assert_deterministic()
```

Comment:
Could we maybe either
1) return the sample=0 per default here or 
2) raise an error which indicates the correct function to call for that behavior